ts-rs/src/export.rs

```diff
@@ -181,9 +166,15 @@ fn generate_decl<T: TS + ?Sized>(out: &mut String) {
 
 /// Push an import statement for all dependencies of `T`
 fn generate_imports<T: TS + ?Sized + 'static>(out: &mut String) -> Result<(), ExportError> {
+    let base = std::env::var("TS_RS_EXPORT_DIR")
+        .ok()
+        .as_deref()
+        .map(Path::new)
+        .unwrap_or_else(|| Path::new("."))
+        .to_owned();
     let export_to =
-        T::get_export_to().ok_or(ExportError::CannotBeExported(std::any::type_name::<T>()))?;
-    let path = Path::new(&export_to);
+        T::EXPORT_TO.ok_or(ExportError::CannotBeExported(std::any::type_name::<T>()))?;
+    let path = base.join(export_to);
```

Comment:
Could we re-use `output_path` here? Might be the case that we have to get rid of the `path::absolute` in `output_path` for that.

Reply:
I don't even think we need to remove `path::absolute`, this path will be fed into `diff_paths`, which uses absolute paths as well

Reply:
you're right, perfect!